src/ui/public/courier/saved_object/saved_object.js

```diff
@@ -1,3 +1,14 @@
+/**
+ * @name SavedObject
+ *
+ * @description SavedObject seems to track a reference to an object in ES,
```

Comment:
I don't think this describes saved_object very well. It's a pretty complex class doing a lot of stuff...